static/spec/openapi.yaml

```diff
@@ -189,14 +193,13 @@ paths:
           schema:
             type: string
           required: true
-        - name: code
-          in: query
-          schema:
-            type: string
-        - name: message
-          in: query
-          schema:
-            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              example: { "message": "not found", "error": 404 }
```

Comment:
Should we be more specific with what the error messages should look like?  Should we make the `message` and `error` fields required?

Reply:
Updated to
```
"requestBody": {
          "required": true,
          "content": {
            "application/json": {
              "schema": {
                "type": "object",
                "required": [
                  "message",
                  "error"
                ],
                "properties": {
                  "message": {
                    "type": "string",
                    "description": "A human-readable error message describing the problem",
                    "example": "The requested resource does not exist"
                  },
                  "error": {
                    "type": "integer",
                    "description": "The HTTP status code of the error",
                    "minimum": 400,
                    "maximum": 599,
                    "example": 404
                  }
```